scripts/src/main/resources/scripts/command/eclipse

```diff
@@ -118,48 +124,42 @@ function doCheckPluginsInstalled() {
 }
 
 function doSetup() {
+  local silent="${1}"
   doDevonCommand java setup silent
-  if [ "${1}" != "silent" ] || [ ! -d "${ECLIPSE_HOME}" ]
+  # check version
+  local javaVersion="${EXTRA_JAVA_VERSION}"
+  if [ -z "${javaVersion}" ]
   then
-    local version="${ECLIPSE_VERSION:-2022-03}"
-    local javaVersion="${EXTRA_JAVA_VERSION}"
-    if [ -z "${javaVersion}" ]
-    then
-      javaVersion="${JAVA_VERSION}"
-    fi
-    if [ -n "${javaVersion}" ]
+    javaVersion="${JAVA_VERSION}"
+  fi
+  if [ -n "${javaVersion}" ]
+  then
+    doVersionCompare "${javaVersion}" "11u0"
+    if [ "${?}" == 2 ]
     then
-      doVersionCompare "${javaVersion}" "11u0"
-      if [ "${?}" == 2 ]
+      # Java version is lower than 11
+      local requireJava11="no"
+      if [ -z "${ECLIPSE_VERSION}" ]
       then
-        # Java version is lower than 11
-        doVersionCompare "${version}" "2020-09"
+        requireJava11="yes"
+      else
+        doVersionCompare "${ECLIPSE_VERSION}" "2020-09"
         if [ "${?}" != 2 ]
         then
           # Eclipse version is >= 2020-09
-          doConfirmWarning "You are using eclipse version ${version} that requires Java 11 or newer.\nHowever, [EXTRA_]JAVA_VERSION is ${javaVersion} - in this setup eclipse can not work.\nPlease update JAVA_VERSION, set EXTRA_JAVA_VERSION or downgrade ECLIPSE_VERSION in your settings/devon.properties!"
+          requireJava11="yes"
         fi
       fi
-    fi
-    doInstall "-" "${ECLIPSE_HOME}" "eclipse" "${version}" "${ECLIPSE_EDITION_TYPE:-java}"
-    if doIsMacOs
-    then
-      if [ ! -x "${ECLIPSE_HOME}/eclipse" ]
+      if [ "${requireJava11}" = "yes" ]
       then
-        echo "Doing workarounds for MacOS quirks..."
-        echo -e "#!/usr/bin/env bash\nECLIPSE_HOME=\"\$(dirname \"\${0}\")\"\n\"\${ECLIPSE_HOME}/Eclipse.app/Contents/MacOS/eclipse\" \$@" > "${ECLIPSE_HOME}/eclipse"
-        chmod a+x "${ECLIPSE_HOME}/eclipse"
+        doConfirmWarning "You are using eclipse version ${ECLIPSE_VERSION:-latest} that requires Java 11 or newer.\nHowever, [EXTRA_]JAVA_VERSION is ${javaVersion} - in this setup eclipse can not work.\nPlease update JAVA_VERSION, set EXTRA_JAVA_VERSION or downgrade ECLIPSE_VERSION in your settings/devon.properties!"
       fi
     fi
-    doAddPlugins
   fi
-  if [ ! -x "${ECLIPSE_HOME}/eclipse" ]
+  doInstall "eclipse" "${ECLIPSE_VERSION}" "${silent}" "${ECLIPSE_EDITION_TYPE:-java}"
```

Comment:
Sadly regarding eclipse one folder in the eclipse download urls changed from "R" to "M2" or "M1", at least for version 2022-12. As an example the download-urls for linux: 
Generated with ide-mirrors: https://ftp.snt.utwente.nl/pub/software/eclipse/technology/epp/downloads/release/2022-12/R/eclipse-java-2022-12-R-linux-gtk-x86_64.tar.gz .
Actual url from utwente.nl:https://ftp.snt.utwente.nl/pub/software/eclipse/technology/epp/downloads/release/2022-12/M2/eclipse-committers-2022-12-M2-linux-gtk-aarch64.tar.gz .
Actual url from eclipse.com: https://archive.eclipse.org/technology/epp/downloads/release/2022-12/M1/eclipse-dsl-2022-12-M1-linux-gtk-x86_64.tar.gz .

Reply:
Yeah, didn't know the meaning of the M before our meeting today. So, the ide should just take the last stable version instead :+1: